src/tweenable.rs

```diff
@@ -1321,10 +1328,19 @@ impl<T> Tweenable<T> for Delay<T> {
         // If completed this frame, notify the user
         if (state == TweenState::Completed) && !was_completed {
             if let Some(user_data) = &self.event_data {
-                events.send(TweenCompleted {
+                let event = TweenCompleted {
                     entity,
                     user_data: *user_data,
-                });
+                };
+
+                // send regular event
+                events.send(event);
+
+                // trigger all global observers
+                commands.trigger(event);
+
+                // trigger all entity-scoped observers
+                commands.trigger_targets(event, [entity]);
```

Comment:
You forgot this one :slightly_smiling_face: 

```suggestion
                // trigger all entity-scoped observers
                commands.trigger_targets(event, entity);
```